app/components/hero-section.tsx

```diff
@@ -1,24 +1,18 @@
-import heroImg from "../imgs/Hero-Image.png";
+import { Image } from './images';
```

Comment:
I moved our images into the `/public` folder — no need to have them bundled together with the JavaScript. The new image component is a wrapper that we can further enhance for image optimizations.